core/src/main/java/bisq/core/provider/price/PriceFeedService.java

```diff
@@ -400,7 +384,12 @@ public void onSuccess(@Nullable Tuple2<Map<String, Long>, Map<String, MarketPric
                 UserThread.execute(() -> {
                     checkNotNull(result, "Result must not be null at requestAllPrices");
                     timeStampMap = result.first;
-                    epochInSecondAtLastRequest = timeStampMap.get("btcAverageTs");
+
+                    // Each currency rate has a different timestamp, depending on when
+                    // the pricenode aggregate rate was calculated
+                    // However, the request timestamp is when the pricenode was queried
+                    epochInMillisAtLastRequest = System.currentTimeMillis();
```

Comment:
Hold on a second, this isn't just changing labels, you're actually changing code here. It looks like you're replacing the timestamp of the actual price data, with the timestamp Bisq updated it from the Pricenode, which is incorrect. This behavior needs to be preserved, for example if a pricenode has an issue and doesn't get data updates, the local Bisq node needs to be able to detect that it's old data and switch to a different Pricenode with recently updated data.

Reply:
Since there is no more BA provider, the `btcAverageTs` field is basically useless.

It is however used here to populate the `epochInMillisAtLastRequest` timestamp -- which is used in the UI to show when the fiat rates were retrieved.

Before, when using BA: this made sense, cause BA was the "single source of truth" for all fiat rates, and Poloniex was the source of alt rates.

Now, when using multiple providers: basically the pricenode itself is the "provider", because it aggregates rates from multiple sources for both fiat and alts.

So, the closest "timestamp" that shows the last time the "provider" (pricenode) was polled, is the one in the code snippet above.

Alternative would be to extract and index every timestamp per exchange rate, then selectively show the right one depending on what currency pair the user selected in the UI. That seems like overkill for what this change tries to achieve. Especially since those "per exchange rate" timestamps are very likely the same, cause they all show the point in time when the avg rate was calculated, and they're all calculated when the pricenode is polled (the avg is freshly calculated based on newest data).

So I would say, this is the simplest and most pragmatic approach (`epochInMillisAtLastRequest` = `timestamp_when_pricenode_last_successfully_queried`).

Reply:
The timestamp is not for when the data was updated from the pricenode, that's irrelevant. The timestamp is for the price itself, i.e. when the price was obtained from the provider, or in the case of a weighted average aka index, the time when the index was last updated. Can you please use the `timestampSec` field for each asset's index?

Reply:
Done